src/fluree/db/ledger/consensus/update_state.clj

```diff
@@ -105,7 +109,7 @@
     ;; publish out new db
     (doseq [[network dbid db-status] init-db-status]
       ;; publish out new db events
-      (event-bus/publish :new-db [network dbid] db-status))))
+      (event-bus/publish :new-ledger [network dbid] db-status))))
 
 
 (defn stage-new-db
```

Comment:
Yeah... my first attempt at this I changed a lot more of these. The rabbit hole went _deep_. I should have stated that my goal here was to deprecate the public API usages of `db` where we meant `ledger` but not every single usage throughout the codebase. Happy to discuss whether or not that _should_ be the goal, but I'm hoping in the main / 2.0 branch it will be easier to change some of these deeper usages when we only need to support the new usage, not both old and new.

Reply:
Oh, ok. Yeah, that makes total sense. I remember trying to do this in one fell swoop a while ago and giving up because it's _a lot_. I'm totally down with sticking to the public api for now.